util/upload-awsbatch-cli.sh

```diff
@@ -76,22 +76,28 @@ main() {
         fi
```

Comment:
isn't this script uploading both clis? why naming it upload-awsbatch-cli?

Reply:
Restored old name